src/parser/csql_grammar.y

```diff
@@ -4865,6 +4879,19 @@ original_table_spec
 			PARSER_SAVE_ERR_CONTEXT ($$, @$.buffer_pos)
 
 		DBG_PRINT}}
+        | DBLINK  '('  dblink_expr ')'   dblink_identifier_col_attrs 
+                {{                       
+                        PT_NODE *ent = parser_new_node (this_parser, PT_SPEC);
+			if (ent)
```

Comment:
```suggestion
                      PT_NODE *ent = parser_new_node (this_parser, PT_SPEC);
			if (ent)
```

Reply:
In the editor, the lines looked good, so I couldn't recognize them.
I'll try to fix it